### DISCUSSION TITLE
How do I get the School enterprise benefits when my school dosnet let me get the emails and I already have an acount

### DISCUSSION THREAD

**Discussion by ghost** [community/community#145335] [GitHub Education]
### Select Topic Area

Question

### Body

pls help

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Ryler4 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your  question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you to the Education category. Good luck!

**Reply by abhinavkothakota** [reply]
Also my university doesn't provide the students with the mail address, only for the teachers. I'm trying to get the student developer pack and I also added one of my extra mail address (contact email) in my account as it said 
Please [add and verify your school-issued email address ](https://github.com/settings/emails)in your account settings – or a contact email if you do not have one. Once your email is verified, refresh this page. I cannot go further stps , what shall I do

**Reply by ghost** [reply]
it wants to verify my school email and I cant  get emails outside the district

**Reply by ghost** [reply]
what do i do??
